src/quacc/recipes/newtonnet/ts.py

```diff
@@ -17,6 +26,11 @@
 except ImportError:
     Sella = None
 
+try:
+    from ase.mep import neb
+except ImportError:
+    neb = None
```

Comment:
I completely got rid of this import because I was not using it.
I think `ase` refactored their directory structure and moved the NEB functions to `ase.mep.neb`. So, I use:
`from ase.mep.neb import NEB`.
Thanks.